apps/frontend/src/app/5_pages/LendPage/components/LendFrame/components/LendFrameChart/LendFrameChart.types.ts

```diff
@@ -1,6 +1,6 @@
 export type ChartData = {
   date: Date;
-  lendAPY: number;
+  lendAPR: number;
```

Comment:
```suggestion
  lendApr: number;
```
Make sure that you use our naming conventions everywhere - capitalize the first letter only.